mailet/standard/src/test/java/org/apache/james/transport/mailets/LogMessageTest.java

```
@@ -0,0 +1,202 @@
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.transport.mailets;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.io.ByteArrayInputStream;
+import java.util.Properties;
+
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.MimeMessage;
+
+import org.apache.mailet.Mail;
+import org.apache.mailet.MailAddress;
+import org.apache.mailet.base.test.FakeMail;
+import org.apache.mailet.base.test.FakeMailContext;
+import org.apache.mailet.base.test.FakeMailetConfig;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.slf4j.Logger;
+
+import com.google.common.base.Charsets;
+import com.google.common.collect.Lists;
+
+public class LogMessageTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    private LogMessage mailet;
+    private FakeMailetConfig mailetConfig;
+    private Logger logger;
+
+    @Before
+    public void setup() {
+        logger = mock(Logger.class);
+        FakeMailContext mailContext = FakeMailContext.builder()
+                .logger(logger)
+                .build();
+        mailetConfig = new FakeMailetConfig("LogContext", mailContext);
+        mailet = new LogMessage();
+    }
+
+    @Test
+    public void getMailetInfoShouldReturnValue() {
+        assertThat(mailet.getMailetInfo()).isEqualTo("LogHeaders Mailet");
+    }
+
+    @Test
+    public void initShouldIgnoreExceptions() throws Exception {
```

ok but it certainly don't ignore all exceptions